refactoring/initial_person_class.py

```diff
@@ -12,11 +12,13 @@ def add_connection(self, person, relation):
         """Add a new connection to a person"""
         if person in self.connections:
             raise ValueError(f"I already know about {person.name}")
+        # didn't know you can have objects as keys but i guess there's no reason
+        # not to
         self.connections[person] = relation
 
     def forget(self, person):
         """Removes any connections to a person"""
-        pass
+        del self.connections[person]
```

Comment:
You could also add a guard clause here so that the user gets a better error message to help them understand what is wrong
```suggestion
        if person not in self.connections:
            raise ValueError(f"I don't know about {person.name}")
        del self.connections[person]
```